src/RouterFactory.php

```diff
@@ -26,7 +26,7 @@ public function __invoke(ContainerInterface $container): RouterInterface
     {
         $factory = $this->engineFactory;
         /* @var $router RouterInterface */
-        $router = $factory($container);
+        $router = $factory();
```

Comment:
What's the reasoning to remove container from callable factory here?

Reply:
> What's the reasoning to remove container from callable factory here?

It not used here 
https://github.com/yiisoft/router-fastroute/blob/master/src/FastRouteFactory.php

Reply:
OK.